interface/HHAnalyzer.h

```diff
@@ -131,6 +143,23 @@ class HHAnalyzer: public Framework::Analyzer {
         std::string m_electron_loose_wp_name;
         std::string m_electron_tight_wp_name;
 
+        // utilities
+        float getCosThetaStar_CS(LorentzVector h1, LorentzVector h2, float ebeam = 6500)
```

Comment:
You should pass a ``const LorentzVector&`` here to avoid a copy at each function call.